lighthouse-core/test/lib/traces/tracing-processor-test.js

```diff
@@ -47,25 +47,6 @@ describe('TracingProcessor lib', () => {
     });
   });
 
-  it('doesn\'t throw when user_timing events have a colon', () => {
```

Comment:
because we don't ask tracing processor to do anything anymore, I can bring back though I guess it doesn't hurt anything